tools/block-generator/README.md

```diff
@@ -156,32 +158,33 @@ Flags:
   -i, --conduit-binary string               Path to conduit binary.
   -l, --conduit-log-level string            LogLevel to use when starting Conduit. [panic, fatal, error, warn, info, debug, trace] (default "error")
       --cpuprofile string                   Path where Conduit writes its CPU profile.
-  -f, --genesis-file string                 file path to the genesis associated with the db snapshot
+  -f, --genesis-file string                 The file path to the genesis associated with the db snapshot.
   -h, --help                                help for runner
   -k, --keep-data-dir                       If set the validator will not delete the data directory after tests complete.
   -p, --metrics-port uint                   Port to start the metrics server at. (default 9999)
   -c, --postgres-connection-string string   Postgres connection string.
-  -r, --report-directory string             Location to place test reports.
-      --reset-db                            If set database will be deleted before running tests.
+  -r, --report-directory string             Location to place test reports. If --times is used, this is the prefix for multiple report directories.
+      --reset-db                            If set the database will be deleted before running tests.
```

Comment:
What do you think about removing this and suggesting the user run `-h` so that we don't have to keep it up to date?

Reply:
316ff045a09e519eea28154ac518f04f5d0cf970